src/app/screens/Home/index.js

```diff
@@ -1,25 +1,129 @@
-import React, { useCallback } from 'react';
+import React from 'react';
 import { connect } from 'react-redux';
+import UTButton from '@widergy/energy-ui/dist/components/UTButton';
 import { push } from 'connected-react-router';
-import { UTButton, UTLabel } from '@widergy/energy-ui';
-import i18 from 'i18next';
+import * as math from 'mathjs';
 
-import { QUOTES } from 'constants/routes';
+import actionCreators from 'redux/history/actions';
+import { HISTORY } from 'constants/routes';
 
 import styles from './styles.module.scss';
 
-const Home = ({ dispatch }) => {
-  const goToQuotes = useCallback(() => dispatch(push(QUOTES)), [dispatch]);
+function Calculator({ dispatch }) {
+  const [expression, setExpression] = React.useState('');
```

Comment:
importate el hook así 

`import React, {useState} from 'react';`

y utilizalo así:

```suggestion
  const [expression, setExpression] = useState('');
```

Reply:
Hecho!